docs/site/sidebars/lb4_sidebar.yml

```diff
@@ -415,6 +414,194 @@ children:
     url: Download-examples.html
     output: 'web, pdf'
 
+- title: 'Connectors reference'
+  url: Connectors-reference.html
+  output: 'web'
+  children:
+
+  - title: 'Memory connector'
+    url: Memory-connector.html
+    output: 'web, pdf'
+
+  - title: 'Database connectors'
+    url: Database-connectors.html
+    output: 'web'
+    children:
+
+    - title: 'Cassandra connector'
+      url: Cassandra-connector.html
+      output: 'web, pdf'
+
+    - title: 'Cloudant connector'
```

Comment:
nitpick: same as https://github.com/strongloop/loopback.io/pull/863#discussion_r309302677, we also support couchdb2. If it takes time to add the new entry we can do it in a separate PR.

Reply:
Let's do it in a separate PR.